pkg/internal/infrastructure/terraform.go

```diff
@@ -111,6 +119,7 @@ func ComputeTerraformerChartValues(
 	if workersCIDR == "" {
 		workersCIDR = config.Networks.Worker
 	}
+	networkConfig["workers"] = workersCIDR
```

Comment:
Do we need this key/value only if `config.Networks.Name == nil`?

Reply:
As we do not consider the `Workers` CIDR to be an optional field, that assignment would always be needed.

Reply:
Yes right, as pointed out in the comment above.